apps/flutter_client_contract_test_service/bin/contract_test_service.dart

```diff
@@ -298,6 +297,61 @@ class TestApiImpl extends SdkTestApi {
     }
   }
 
+  LDContext _buildContextForComparison(SingleOrMultiBuildContext input) {
```

Comment:
Contexts for comparison are not built the same was as contexts for conversions or context builds. We may need to adjust them as well, but they have different considerations so the various types are different.